staticd/static_routes.c

```diff
@@ -360,10 +360,9 @@ void static_install_nexthop(struct static_nexthop *nh)
 	switch (nh->type) {
 	case STATIC_IPV4_GATEWAY:
 	case STATIC_IPV6_GATEWAY:
-		static_zebra_nht_register(nh, true);
-		break;
 	case STATIC_IPV4_GATEWAY_IFNAME:
 	case STATIC_IPV6_GATEWAY_IFNAME:
+		static_zebra_nht_register(nh, false);
```

Comment:
Isn't possible to do this operation only if the source IP is different from the previous one (last used)?